.vsts-ci.yml

```diff
@@ -190,12 +191,20 @@ jobs:
       $ROOT_DOCKER_FOLDER/.ci/setup.sh || exit -1
       $ROOT_DOCKER_FOLDER/.ci/test.sh || exit -1
       EOF
+      IMAGE_URI="quay.io/pypa/manylinux2014_${ARCH}"
+      docker pull "${IMAGE_URI}" || exit -1
+      PLATFORM=$(
+        docker image inspect "${IMAGE_URI}"\
+        | jq -r '.[0] | "\(.Os)/\(.Architecture)"'
+      ) || exit -1
+      echo "detected image platform: ${PLATFORM}"
```

Comment:
Would you prefer leaving this debugging message?

Reply:
yes definitely. It will be helpful if this job fails in the future with cryptic errors, since there's no other place in the logs to see the value of `--platform` that was used.